base/src/main/scala/typeclass/Bind.scala

```diff
@@ -3,7 +3,7 @@ package typeclass
 
 trait Bind[M[_]] {
   def apply: Apply[M]
-  def flatMap[A, B](ma: M[A])(f: A => M[B]): M[B]
+  def flatMap[A, B](ma: => M[A])(f: A => M[B]): M[B]
```

Comment:
...this is because the usefully lazy signature seems to be

```scala
[A, B](ma: M[A])(f: (=> A) => M[B]): M[B]
```

but that seems annoying for other reasons.